sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -69,6 +61,14 @@ EXPORT_SYMBOL_GPL(snd_soc_acpi_intel_tgl_machines);
 
 /* this table is used when there is no I2S codec present */
 struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_sdw_machines[] = {
+	{
+		.id = "10EC1308",
+		.drv_name = "sdw_rt711_rt1308_rt715",
+		.link_mask = 0x1, /* RT711 on SoundWire link0 */
+		.links = tgl_i2s_rt1308,
+		.sof_fw_filename = "sof-tgl.ri",
+		.sof_tplg_filename = "sof-tgl-rt711-i2s-rt1308.tplg",
+	},
```

Comment:
that doesn't seem like a good idea, and clearly this contradicts the comment above. This also changes the search on the the platform side, where you look first for an I2S codec, before looking at SoundWire.

Reply:
Now we want to support tgl_i2s_rt1308 in sdw_rt711_rt1308_rt715 machine driver and discard the original sdw_rt711_i2s_1308. So we need to select sdw machine driver for this case.

Reply:
I discard the change and only change machine driver and topology name.